spec/support/mockPartnerConfig.json

```diff
@@ -2,27 +2,26 @@
     "timeout": 1000,
     "xSlots": {
         "1": {
-            "placementId": "12913056",
+            "placementId": "54321",
             "sizes": [
-                [320, 51],
-                [123, 321]
+                [300, 250]
             ]
         },
         "2": {
-            "placementId": "12913056",
+            "placementId": "12345",
             "sizes": [
-                [300, 250]
+                [300, 600]
             ]
         },
         "3": {
-            "placementId": "12913056",
+            "placementId": "654321",
             "sizes": [
-                [320, 20]
+                [728, 90]
             ]
         }
     },
     "mapping": {
-        "320x50_300x250": ["1", "2"],
-        "320x20": ["3"]
+        "728x90_970x250": ["1", "2"],
```

Comment:
Currently xSlot 1,2 have size 300x250 and 300x600 and is mapped to a 728x90 and 970x250 slot. xSlot 3 with size 728x90 and is mapped to a 300x250 slot. Can you please change the size and ensure the placementId in the mock config actually return a valid bid? 

Currently with the wrapper in the test-wrapper branch https://github.com/indexexchange/playground-xyz-adapter/blob/test-wrappers/playground-xyz-adapter-wrapper-2018-6-20%2022:5.js, your end point return a 204 no-content response and there's some issues with `Access-Control-Allow-Origin`.